pages/api/events.js

```diff
@@ -12,14 +12,29 @@ export default async function handler(req, res) {
   return res.status(200).json(events);
 }
 
-export async function getEvents() {
+export async function getEvents(inMap = false) {
```

Comment:
I recommend changing this to be less about the feature, as we might filter by it on the events page in the future, what about something like `withLocation`?